project-settings/teams-and-users/manage-roles.md

```diff
@@ -86,6 +86,25 @@ _Collection permissions_ allow you to enable/disable the following collection-sp
 - _Delete_: delete a record of this collection
 - _Export_: export the list of records of this collection
 
+### Default permissions <a href="#default-permissions" id="default-permissions"></a>
+
+_Default permissions_ allow you to choose default permissions for each newly created Collection or Smart Action.
+
+
+Here is the initial configuration for a new role:
+* Smart Actions
+  |Trigger| Require approval  | Approve| Self approve| 
+  | :-: | :-: | :-: | :-: | 
+  | :heavy_check_mark: | - | - | - | - |
+* Collections:
+  | Read (list) | Read (details) | Create | Update | Delete | Export |
+  |  :-: |  :-: |  :-: |  :-: |  :-: |  :-: |
+  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | 
```

Comment:
The end result does not look great
![Screenshot 2023-09-14 at 08 38 55](https://github.com/ForestAdmin/documentation-user-guide/assets/664261/1f723c3e-922c-4ef4-b389-1b506e9cf737)

You can access the preview by clicking on GitHub checks details related to GitBook. 

1. We should remove the list style for Smart Actions and Collections.
2. Maybe the horizontal display idea like in the app is not a great idea, and so we should go vertical to prevent horizontal scroll.

Reply:
It looks much better in a vertical table